src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest/PublishVerticalManifest.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest
+{
+    public class PublishVerticalManifest : Task
+    {
+        [Required]
+        public ITaskItem[] AssetManifestPaths { get; set; }
+        [Required]
+        public string VerticalAssetManifestLocalPath { get; set; }
```

Comment:
I think the `OutputPath` makes more sense here.

```suggestion
        public string VerticalAssetManifestOutputPath { get; set; }
```